content/en/docs/instrumentation/python/automatic.md

```diff
@@ -88,6 +92,15 @@ The examples that follow send instrumentation results to the console. Learn more
 about installing and configuring the [OpenTelemetry Distro](../distro) to send
 telemetry to other destinations, like an OpenTelemetry Collector.
 
+**Note**: If you were to use auto-instrumentation and manually instrumenting your code the
+pacakage `opentelemetry-distro` won't let you set a `Resource` with attributes defined in your code, 
+you will need to set all your desired parameters through [environment variables][env]
```

Comment:
What if you include the SDK package? Or are entities treated as entirely different when you manually instrument + use the agent?